api/controllers/WebViewController.js

```diff
@@ -349,9 +354,9 @@ module.exports = {
     }`;
 
     const twitterShareBody =
-      'A text that sends you a daily self-care advice? Yes, please. Join me %26 sign up for Shine! %23ShineOn';
+      'A text that sends you daily self-care advice? Yes, please. Join me %26 sign up for Shine! %23ShineOn';
```

Comment:
Do you think it would be worth it to pull the encoded values from an object
`cont obj = { '@':  '%23' , ...}`

Reply:
For something trivial like this, I don't think so. And if I'm understanding your suggestion correctly, I imagine it'd also need some additional code to then parse and encode before rendering to the final page.